rotkehlchen/data_migrations/migrations/migration_11.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import ChainID
+
+if TYPE_CHECKING:
+    from rotkehlchen.data_migrations.progress import MigrationProgressHandler
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def _get_optimism_transaction_fees(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgressHandler') -> None:  # noqa: E501
+    """Since we now need 1 extra column per optimism transaction we need to repull all
+    optimism transactions to store the l1 gas fee"""
+    with rotki.data.db.conn.read_ctx() as cursor:
+        cursor.execute('SELECT identifier, tx_hash FROM evm_transactions WHERE chain_id=10')
+        txhash_to_id = {txhash: identifier for identifier, txhash in cursor}
+
+    progress_handler.set_total_steps(len(txhash_to_id))
+
+    db_tuples = []
+    optimism_manager = rotki.chains_aggregator.get_evm_manager(ChainID.OPTIMISM)
```

Comment:
It happened in previous releases that if you pull a database that needs to be updated the `chains_aggregator` attribute at rotki is not set yet when the update is executed

Reply:
> It happened in previous releases that if you pull a database that needs to be updated the `chains_aggregator` attribute at rotki is not set yet when the update is executed

What did we do then? I don't see this in any other migration that's left.